### ISSUE TITLE
6th Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#7]
### 진행 전
- 배치(Batch) 6.8% 축소, 에폭(Epoch) 1000회로 증가, 학습률(η)을 0.005로 증가
- 텐서보드 기능추가(손실율 그래프 및 알고리즘 시각화 목적)
![NNN mk3 Tensorboard](https://user-images.githubusercontent.com/47406388/69921869-166eaf00-14da-11ea-8de2-decaf3033516.png)
### 진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현
- Softmax로 확률 구현
- Adam으로 Optimizer 구현
- 배치사이즈와 학습률을 두 가지 경우로 개별 학습
- 에폭-손실(Loss) 곡선과 알고리즘을 텐서보드로 저장
- 훈련 파일을 ckpt 확장자로 저장 후 시험
### 진행 후
- 에폭-손실 곡선
![7th Epoch-Loss Rate](https://user-images.githubusercontent.com/47406388/69921872-238b9e00-14da-11ea-9b05-917c29d8e84a.jpg)
- 기존 신경망에 비해 손실율이 보다 0에 가깝게 기록되어 학습이 개선됨
[7th Learning.txt](https://github.com/roqhdehd502/Franklin-Detecter/files/3908832/7th.Learning.txt)
- 정확도(Acc)가 92.86%로 증가, 예전학습에 비해 약 41.84% 개선됨
- 학습률과 배치를 좀 더 보정하면 정확도가 좀 더 오를것으로 기대함
